_posts/2017-12-19-2017-year-in-review.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: post
+title: 'Apache Flink in 2017: Year in Review'
+date: 2017-12-19T09:00:00.000Z
+author: Mike Winters
+excerpt: As 2017 comes to a close, let's take a moment to look back on the Flink community's great work during the past year.
+categories: news
+---
+
+2017 was an exciting year for the Apache Flink® community, and the [release of Flink 1.0 in March](http://flink.apache.org/news/2016/03/08/release-1.0.0.html) marked the first time in Flink's history that the community guaranteed API backward compatibility for all versions in a series. This step forward for Flink was followed by many new and exciting production deployments in organizations of all shapes and sizes, all around the globe.
```

Comment:
@wints This needs a rewrite, but I'm not entirely sure what to add as the 'banner' news.

Reply:
Here's a proposal @ChrisChinchilla: 

2017 was another exciting year for the Apache Flink® community, with 3 major version releases ([Flink 1.2.0 in February](http://flink.apache.org/news/2017/02/06/release-1.2.0.html), [Flink 1.3.0 in June](http://flink.apache.org/news/2017/06/01/release-1.3.0.html), and [Flink 1.4.0 in December](http://flink.apache.org/news/2017/12/12/release-1.4.0.html)) and the first-ever [Flink Forward in San Francisco](https://sf-2017.flink-forward.org/), giving Flink community members in another corner of the globe an opportunity to connect. Users shared detail about many innovative production deployments, redefining what is possible with a modern stream processing framework like Flink.